spec/controllers/application_controller_spec.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 require 'rails_helper'
+require 'openssl'
```

Comment:
I don't think this import is used, I think we can drop it?

Reply:
deleted